content/blog/2019-10-22-react-release-channels.md

```diff
@@ -1,62 +1,62 @@
+**El único canal de lanzamiento oficialmente compatible para aplicaciones de cara al usuario es el último**. Las versiones siguiente y experimental se proporcionan solo con fines de prueba, y no ofrecemos garantías de que el comportamiento no cambie entre lanzamientos. No siguen el protocolo semver que usamos para las últimas versiones.
 
-By publishing prereleases to the same registry that we use for stable releases, we are able to take advantage of the many tools that support the npm workflow, like [unpkg](https://unpkg.com) and [CodeSandbox](https://codesandbox.io).
+Publicando prelanzamientos en el mismo registro que usamos para lanzamientos estables, podemos aprovechar las muchas herramientas que admiten el flujo de trabajo npm, me gusta [unpkg](https://unpkg.com) and [CodeSandbox](https://codesandbox.io).
 
-### Latest Channel {#latest-channel}
+### Último Canal {#último-canal}
 
-Latest is the channel used for stable React releases. It corresponds to the `latest` tag on npm. It is the recommended channel for all React apps that are shipped to real users.
+Latest es el canal utilizado para las versiones estables de React. Corresponde a la etiqueta `latest` en npm. Es el canal recomendado para todas las aplicaciones de React que se enviado a usuarios reales.
 
-**If you're not sure which channel you should use, it's Latest.** If you're a React developer, this is what you're already using.
+**Si no está segura de qué canal usar, es Latest.** Si eres una desarrolladora React, esto es lo que ya estas usando.
 
-You can expect updates to Latest to be extremely stable. Versions follow the semantic versioning scheme. Learn more about our commitment to stability and incremental migration in our [versioning policy](/docs/faq-versioning.html).
+Puede esperar que las actualizaciones de Latest sean extremadamente estables. Las versiones siguen la semántica esquema de versionado. Obtenga más información sobre nuestro compromiso con la estabilidad y la migración incremental en nuestro [versioning policy](/docs/faq-versioning.html).
 
-### Next Channel {#next-channel}
+### Siguiente-canal {#siguiente-canal}
 
-The Next channel is a prerelease channel that tracks the master branch of the React repository. We use prereleases in the Next channel as release candidates for the Latest channel. You can think of Next as a superset of Latest that is updated more frequently.
+El Siguiente canal es un canal de presentación que pistas la rama maestra del repositorio de React. Usamos pre lanzamientos en el canal Siguiente como candidatos de lanzamiento para el Latest canal. Puede pensar en Next como un superconjunto de Latest que se actualiza con más frecuencia.
```

Comment:
@aroundarmor this is one of the parts that need a better translation. If you proof read the whole document you'll find many occurrences like this one.